### ISSUE TITLE
active_record_validations.html should be clear about return values of custom validation methods

### ISSUE THREAD

**Issue by benjiwheeler** [rails/rails#20792]
The "Custom Methods" section doesn't make any mention of the return values of custom validation functions. It only shows the errors array being appended to indicate validation failure.

If "return true" and "return false" have absolutely no meaning for custom validation functions, it should state that clearly, because that is a very common mistake to make.

If they *do* have meaning, it should state *that* clearly.

**Comment by senny** [comment]
@benjiwheeler I added a paragraph to the API and linked there from the guide. Thanks for reporting.

**Comment by benjiwheeler** [comment]
Thanks for fast turnaround. Reading over guide/docs, I think the problem was that the wording of the "valid?" section is hard to parse... "Rails uses the 'valid?' method" seemed like it meant that this was a capability of Rails, when it really means that *all* ActiveModel/ActiveRecord objects get valid? called on them by Rails.

**Comment by senny** [comment]
@benjiwheeler if you have suggestions to make the guides more clear, a PR is always welcome. You can find the guide source in the rails/rails repo: https://github.com/rails/rails/tree/master/guides/source

**Comment by benjiwheeler** [comment]
Thanks, I created PR https://github.com/rails/rails/pull/20795
